### PR TITLE
Add Password Patterns and Password History tutorials

### PR DESCRIPTION
**Pull request by YvonneW**
Added the following `.md` files:

- docs/learn/configuring-password-patterns.md
- docs/learn/configuring-password-policies.md
- docs/learn/validating-password-history.md
	
Added the following `.png` files associated with the above `.md` files:
- docs/assets/img/learn/login-permission.png
- docs/assets/img/learn/password-history-form.png
- docs/assets/img/learn/password-history-option.png
- docs/assets/img/learn/password-history.png
- docs/assets/img/learn/password-patter-violation-error-message.png
- docs/assets/img/learn/password-pattern.png
- docs/assets/img/learn/password-patterns-form.png
- docs/assets/img/learn/password-patterns-option.png
- docs/assets/img/learn/passwore-history-error-message.png
- docs/assets/img/learn/role-permissions-option.png
- docs/assets/img/learn/sign-in-form.png
- docs/assets/img/learn/user-account.png
- docs/assets/img/learn/view-roles-option.png
- docs/assets/img/learn/change-password-form.png
- docs/assets/img/learn/change-password-option.png
- docs/assets/img/learn/consent-form.png

Updated the following `.md` files: 
- mkdocs.yml
- docs/learn/managing-user-accounts.md

### CODE REVIEW

File: en/docs/learn/configuring-password-patterns.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Configuring Password Patterns
+
+WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and enforce them at the point of user creation. This enables complying with password policies deem necessary through various regulations. WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and enforce them at the point of user creation. This enables complying with password policies deem necessary through various regulations.
```
Comment (gomathyk) [inline en/docs/learn/configuring-password-patterns.md]: and enforcing them

File: en/docs/learn/configuring-password-patterns.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Configuring Password Patterns
+
+WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and enforce them at the point of user creation. This enables complying with password policies deem necessary through various regulations. WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and enforce them at the point of user creation. This enables complying with password policies deem necessary through various regulations. 
+
```
Comment (sherenewso2) [inline en/docs/learn/configuring-password-patterns.md]: WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and **enforcing** them at the point of user creation. This enables complying with password policies **that are deemed** necessary through various regulations. 

This sentence is repeated twice.

File: en/docs/learn/configuring-password-patterns.md (hunk 1 above)
Comment (gomathyk) [reply en/docs/learn/configuring-password-patterns.md]: This whole thing has been repeated somehow. 
"WSO2 Identity Server enables defining custom [password policies](../../learn/configuring-password-policies) and enforce them at the point of user creation. This enables complying with password policies deem necessary through various regulations. "

File: en/docs/learn/configuring-password-patterns.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+<img src="../../assets/img/learn/password-pattern.png" alt="Password Pattern" width="700">  
+
+## Scenario
+
+**Pickup** is a cab company that has many employees who use different credentials to sign in to different internal enterprise applications. Robert who is an administrator at Pickup wants enforce the following password pattern when logging in.
```
Comment (gomathyk) [inline en/docs/learn/configuring-password-patterns.md]: wants to enforce

File: en/docs/learn/configuring-password-patterns.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
```
Comment (sherenewso2) [inline en/docs/learn/configuring-password-patterns.md]: Robert{comma} who is an administrator at Pickup {comma} wants **to** enforce the following password pattern when logging in.

File: en/docs/learn/validating-password-history.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
+#Validating Password History
+
+WSO2 Identity Server enables restricting the user from re-entering recently used passwords as a new password. For example, if we have set this configuration to 5, the user will not be able to re-use any of the five most recently used passwords when setting a new password.
+
+<img src="../../assets/img/learn/password-history.png" alt="Password History" width="700">
+
+## Scenario
+
+**Pickup** is a cab company that has many employees who use different credentials to sign in to different internal enterprise applications. Robert who is an administrator at Pickup wants to restrict users from re-using the three most recently used passwords when setting a new password.
```
Comment (sherenewso2) [inline en/docs/learn/validating-password-history.md]: Robert{comma} who is an administrator at Pickup {comma} wants to...

File: en/docs/learn/validating-password-history.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+		<img src="../../assets/img/learn/passwore-history-error-message.png" alt="Passwrod History Validation error message" width="600" style="border:1px solid grey">
+
+	7.	To mimic three consecutive password changes, change Larry's password to the following sequentially. 
+
+		-	`test123`
+		-	`test234`
+		-	`test345`
+
+		Note that those passwords gets successfully added to the system.
```
Comment (sherenewso2) [inline en/docs/learn/validating-password-history.md]: Note that these passwords get added to the system successfully.

File: en/docs/learn/validating-password-history.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+3.	To change the password:
+
+	1.	Access WSO2 Identity Server Dashboard at `https://localhost:9443/dashboard/`.
+
+	2.	Log in with the credentials of the user account that you created.
+
+		<img src="../../assets/img/learn/sign-in-form.png" alt="Sign In form" width="400" style="border:1px solid grey"> 	
+
+	3.	Consent to share the shown attributes with the given service provice provider.
```
Comment (gomathyk) [inline en/docs/learn/validating-password-history.md]: service provider.

File: en/docs/learn/configuring-password-patterns.md (hunk 2 above)
Comment (YvonneW) [reply en/docs/learn/configuring-password-patterns.md]: Fixed

File: en/docs/learn/configuring-password-patterns.md (hunk 4 above)
Comment (YvonneW) [reply en/docs/learn/configuring-password-patterns.md]: Fixed

**Pull request merged**